Misc/NEWS.d/next/Core_and_Builtins/2025-11-10-23-07-06.gh-issue-141312.H-58GB.rst

```diff
@@ -0,0 +1,2 @@
+Restrict passing of non-integer object to the ``longrangeiter.__setstate__``.
+Patch by Sergey Miryanov.
```

Comment:
This mostly refers to implementation details. From the user's point of view, the bug manifested as an assertion failure, now it is gone.

"Fix assertion failure in the `__setstate__` method of the range iterator if non-integer argument was passed." Or something like.